### PR TITLE
wrote tests for cropping, small fixes

### PR DESCRIPTION
**Pull request by santient**
Note: some tests may be incomplete as the code I was testing is uncommented

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ cache: pip
 install:
   - pip install -r requirements.txt
 script:
-  - pytest tests/
\ No newline at end of file
+  - pytest -v
```
Comment (jccaicedo) [inline .travis.yml]: Add new line.

File: deepprofiler/imaging/cropping.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def build_augmentation_graph(self):
 
         # Outputs and queue of the data augmentation graph
         augmented_op = deepprofiler.imaging.augmentations.augment_multiple(
-            self.input_variables["labeled_crops"][0],
+            tf.cast(self.input_variables["labeled_crops"][0], tf.float64),
```
Comment (jccaicedo) [inline deepprofiler/imaging/cropping.py]: Can you make it work with `tf.float32` instead?

File: tests/dataset/test_indexing.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def test_write_compression_index():
     open_file = open("deepprofiler/examples/config/test_indexing.json")
     config = json.load(open_file)
     deepprofiler.dataset.indexing.write_compression_index(config)
-    test_output = pd.DataFrame.from_csv("deepprofiler/examples/metadata/tmp/index.csv")
+    test_output = pd.read_csv("deepprofiler/examples/metadata/tmp/index.csv", index_col=0)
```
Comment (jccaicedo) [inline tests/dataset/test_indexing.py]: Good idea!

**Comment by santient** [comment]
Fixed float issue.

**Pull request merged**